app/assets/javascripts/admin/modules/navbar-toggle.js

```diff
@@ -0,0 +1,27 @@
+window.GOVUK = window.GOVUK || {}
```

Comment:
I'm a bit unclear why this extra JS is added the commit doesn't explain much.

Is it definitely needed in this PR (i.e. do things break without it) as it's already quite an intense amount of changes.

Reply:
So this is something that I wanted to split up but was requested that we should remove all legacy admin JS and rewrite the navigation toggle in vanilla JS